src/context/GlobalContextProvider.tsx

```diff
@@ -22,15 +33,18 @@ export const GlobalContextProvider: React.FC = ({ children }: { children: any })
     return <p>Error loading coverId {coverId}!</p>;
   }
 
+  if (!isChaptersSuccess) {
+    return <p>Error loading chapters for coverId {coverId}!</p>;
```

Comment:
I think that in the future we should make a component like Notification, which we can style as Error, or as Info.